spec/basic/expected_conditions_spec.js

```diff
@@ -0,0 +1,167 @@
+var EC = protractor.ExpectedConditions;
+
+describe('expected conditions', function() {
+  beforeEach(function() {
+    browser.get('index.html#/form');
+  });
+
+  it('should have alertIsPresent', function() {
+    var alertIsPresent = EC.alertIsPresent();
+    expect(alertIsPresent.call()).toBe(false);
```

Comment:
I agree that we should optimize for being used in `browser.wait`. 

I'm fine with either `call()` or just `()`, so you can leave as is.